docs/tex/api/reference.tex

```diff
@@ -0,0 +1,65 @@
+\title{Reference}
+
+{{navbar}}
+
+\subsubsection{Models}
+
+Most edward models are wrappers around tensorflow distrbutions. Therefore, the
```

Comment:
+ `Most edward models` -> `Most Edward random variables`
+ capitalize Edward, TensorFlow;
+ `distrbutions` -> `distributions`
+ `Therefore, the` -> `Therefore the`